src/main/java/frc/robot/commands/TeleopClawActuationCommand.java

```diff
@@ -0,0 +1,30 @@
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.lib.DTXboxController;
+import frc.robot.subsystems.grabber.GrabberClaw;
+
+public class TeleopClawActuationCommand extends CommandBase{
+    private DTXboxController controller;
+    private GrabberClaw claw;
+
+    @Override
+    public void execute(){
+        boolean aButton = controller.getAButtonPressed();
+        //close for cone ^
+        boolean bButton = controller.getBButtonPressed();
+        //close for cube ^
+        boolean yButton = controller.getYButtonPressed();
+        //open all the way ^
+        if (aButton){
+            claw.closeClaw(true);
+        }
+        else if (bButton){
+            claw.closeClaw(false);
+        }
+        else if (yButton){
+            claw.openClaw();
+        }
```

Comment:
I’d like to see two separate methods in the `Claw` class, one to open and close the claw, and another to select the pressure. This would better represent the solenoids on the physical robot. The boolean flag is just confusing to read.

Reply:
File is gone :(